ocaml/idl/datamodel_pool.ml

```diff
@@ -409,17 +409,40 @@ open Datamodel_types
       ~name:"certificate_install"
       ~doc:"Install a TLS CA certificate, pool-wide."
       ~params:[String, "name", "A name to give the certificate";
-               String, "cert", "The certificate"]
+               String, "cert", "The certificate in PEM format"]
       ~allowed_roles:_R_POOL_OP
+      ~internal_deprecated_since:rel_next
```

Comment:
Please use `~lifecycle` to replace `~in_oss_since`, `~in_product_since` and `~internal_deprecated_since`. This allows you to provide information about the deprecation, e.g. that the function has been replaced by another one.